Description: fixing nodejs scripts for Debian
Author: Markus Frosch <markus@lazyfrosch.de>
Last-Update: 2012-11-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/icinga-web/inGraph/lib/nodejs/ingraph-renderer.js
+++ b/icinga-web/inGraph/lib/nodejs/ingraph-renderer.js
@@ -1,4 +1,4 @@
-#!/usr/bin/env node
+#!/usr/bin/nodejs
 
 /**
  * inGraph renderer
